src/tests/test_array.py

```diff
@@ -143,7 +143,7 @@ def testArrayContains(self):
 
         self.assertFalse(5 in items)    # "H:\Python27\Lib\unittest\case.py", line 592, in deprecated_func,
         self.assertFalse(-1 in items)   #TypeError: int() argument must be a string or a number, not 'NoneType'
-        self.assertFalse(None in items) # which threw ^ here which is a little odd.
```

Comment:
What's changed with the result of this test? This test seems valid and so should still pass as far as I can tell.

I can see that someone would argue that it should raise a TypeError. If that's what's happening we either need to a) preserve the existing behaviour so as not to break existing code or b) make a decision to change it and update the test to reflect that.